src/include/function/list/functions/list_resolution_function.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "binder/expression/expression_util.h"
+#include "function/function.h"
+
+namespace kuzu {
+namespace function {
+
+static inline void resolveEmptyList(const ScalarBindFuncInput& input,
```

Comment:
Let's write these two function into a class called `ListFunctionUtils`. Because these are static functions defined in header files, if not wrapped by function, it may potentially conflict with other function declarations in other files. For example, `resolveNulls` sounds like it will work for any function type.

Also move implementation to a cpp file. Let's keep the header small.